packages/kbn-guided-onboarding/src/components/landing_page/guide/guide_card.tsx

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { Fragment, useCallback, useState } from 'react';
+
+import {
+  EuiCard,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiTextColor,
+  useEuiTheme,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { css } from '@emotion/react';
+// import { toMountPoint } from '@kbn/react-kibana-mount';
+import { DeploymentDetailsModal, DeploymentDetailsProvider } from '@kbn/cloud/deployment_details';
+import type { ToMountPointParams } from '@kbn/react-kibana-mount';
+import { MountPoint } from '@kbn/core-mount-utils-browser';
+import ReactDOM from 'react-dom';
+import { GuideState } from '../../../types';
+import { GuideCardConstants } from './guide_cards.constants';
+import { GuideCardsProps } from './guide_cards';
+
+const toMountPoint = (node: React.ReactNode, params: ToMountPointParams): MountPoint => {
```

Comment:
I'm not too familiar with @kbn/react-kibana-mount... but is there a reason why you're no longer using `toMountPoint` from the package?

Reply:
I was having issues with the context in https://github.com/elastic/kibana/pull/171586#issuecomment-1839228379. Wrapping it in a Fragment instead didn't impact the presentation of the modal and removed the document is not defined 😌